Hunting Queries/LAQueryLogs/QueryForSecrets.yaml

```diff
@@ -0,0 +1,24 @@
+id: 2bf19f27-0466-4c16-a821-ce84e524476d
+name: Queries looking for secrets
+description: |
+  'This hunting query looks for queries that appear to be looking for secrets or passwords in tables.'
+requiredDataConnectors:
+  - connectorId: CustomConnector
```

Comment:
Lets use connectorId: "AzureMonitor(Query Audit)" for now, it may change slightly but should be under Azure Monitor.